src/quick-lint-js/parse.h

```diff
@@ -2808,7 +2808,16 @@ class parser {
 
     if (this->peek().type == token_type::kw_else) {
       this->skip();
+      const char8 *end_of_else = this->lexer_.end_of_previous_token();
+      bool has_left_paren = this->peek().type == token_type::left_paren;
       parse_and_visit_body();
+      bool has_left_curly = this->peek().type == token_type::left_curly;
+      if (this->peek().has_leading_newline == false && has_left_paren &&
+          has_left_curly) {
+        this->error_reporter_->report(error_missing_if_after_else{
```

Comment:
Nit: Include a comment with an example code snippet.
```suggestion
        // if (cond) {} else (cond) {}  // Invalid.
        this->error_reporter_->report(error_missing_if_after_else{
```